app/Controllers/Http/ExplorerController.js

```diff
@@ -1484,14 +1484,16 @@ class ExplorerController {
     const ws = await this.getWorkspace();
     const PATH_CONFIG = path.join(ws.toString(), '/TFSettings.json');
     const cfg = await this.getJsonConfig(PATH_CONFIG);
-    const COMMAND_FILES_PATH = Env.get('COMMAND_FILES_PATH', path.join(Env.get('ROOT_PATH'), 'DeepLearning'))
```

Comment:
@petrovfedor please check if this is the correct state of code (according to your changes)

Reply:
Seems OK to me